build.py

```diff
@@ -30,10 +31,13 @@
 DEFAULT_CONFIG = "etc/kapacitor/kapacitor.conf"
 PREINST_SCRIPT = None
 
+# Default AWS S3 bucket for uploads
+DEFAULT_BUCKET = "kapacitor"
```

Comment:
I wasn't aware of that. Regardless, I've created a kapacitor bucket that we can use for releases or nightlies (or you can continue to use influxdb). I've been trying to spread out S3 uploads into separate buckets, since the influxdb bucket is becoming a bit unwieldy.